app/controllers/request-to-go-live/organisation-address/post.controller.test.js

```diff
@@ -177,7 +179,7 @@ describe('organisation address post controller', () => {
       })
     })
 
-    describe('view page when `Stripe setup`', () => {
+    describe('view page when `setting up a new Stripe gateway account`', () => {
```

Comment:
Can we simplify descriptions as it is confusing to understand the tested functionality?

It is a `post-op` and `view page` in the description implies it is a get. There is another suite `Setup new Stripe account` which looks similar to this. Can these be merged?

Noticed that some tests here are starting with `it('when the required`. Can these be updated for readability? Could be something like:

`it('should return errors, when the required fields are missing')`